deepfence_agent/store/updatemon/src/main.rs

```diff
@@ -0,0 +1,475 @@
+use std::{
```

Comment:
nit: we could break down this file into multiple ones, having `supervisrod` & `artifact` modules would makes thing more organized